tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -646,6 +646,24 @@ const reducer = createReducer(
       },
     };
   }),
+  on(actions.metricsChangeCardWidth, (state, {cardMaxWidthInVW}) => {
```

Comment:
(Dunno how to comment lines above, so adding it here) On line 398, do we need to have
```
if (typeof partialSettings.timeSeriesCardMaxWidthInVW === 'number') {
      metricsSettings.timeSeriesCardMaxWidthInVW = partialSettings.timeSeriesCardMaxWidthInVW;
    }
```
like the other ones?

Reply:
Unfortunately we didn't preserve the card width setting (yet) so it's meaningless to add them here.